activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -0,0 +1,389 @@
+# frozen_string_literal: true
+
+begin
+  require "redis"
+  require "redis/distributed"
+rescue LoadError
+  warn "The Redis cache store requires the redis gem. Please add it to your Gemfile."
+  raise
+end
+
+# Prefer the hiredis driver but don't require it.
+begin
+  require "redis/connection/hiredis"
+rescue LoadError
+end
```

Comment:
Maybe it's dubious. Maybe it's good practice. Maybe it's …

Reply:
@jeremy Do I need to explicitly add `gem 'hiredis-client'` in my gemfile to use the `hiredis` as a driver in Redis cache store?

Reply:
Yes @MuhammadRazaAslam